### PR TITLE
Simplify GitHub Teams

### PR DESCRIPTION
**Pull request by philschatz**
I pulled all the Team info for the openstax org in GitHub and tried to come up with an alternative (probably flawed, let me know!). I'm suggesting a simpler structure using the not-so-new Teams stuff in GitHub ( https://github.com/blog/2064-new-organization-permissions-now-available and https://github.com/orgs/improved-permissions )

In the new GitHub Teams system: an ​_admin_​ is a ​**role**​ given to certain users in each repository and a team is a group of people that have write permissions to a repository (or read/write in the case of private repos).

Basically, I'm suggesting we have 4 teams for access-control and several "teams" for communication (`@mentioning` people):

- `all`: write to public repos + `oer.exports`, and maybe read to `*-deployment` repos?
- `biglearn`: write to biglearn repos
- `research`: write to research/labs repos
- `devops`: write to deployment repos

Also, for an example of non-coders using GitHub: https://github.com/blog/2093-how-the-services-team-uses-github

See the files in this PR for the old Teams and which repos/users would have access to the new teams

### CODE REVIEW

**Comment by cnuber** [comment]
There will need to be a team that has read access to deployment.  While sensitive items are encrypted, only certain developers who actively contribute to deployment scripts would need access to the repo and allowing * is not in best security practices for a private repo that somewhat exposes our network infrastructure and settings.  DevOps goal is to provide developers with push-button access for deployments via RunDeck which will eliminate the need for them to checkout the repo to deploy as well.

**Comment by philschatz** [comment]
Thanks! Updated proposal to have `devops-read` and `devops-write` :smile:
